### ISSUE TITLE
size parameter doesn't work for the trades rest API call

### ISSUE THREAD

**Issue by WillCalderwood** [okcoin-okex/API-docs-OKEx.com#58]
Changing size has no effect.

example requests.
This should return 100 trades:
https://www.okex.com/api/v1/trades.do?symbol=eos_btc&since=0&size=100

This should return 1 trade:
https://www.okex.com/api/v1/trades.do?symbol=eos_btc&since=0&size=1

Both return 60 trades.

**Comment by rengelking** [comment]
since parameter doesn't seem to work either

**Comment by WillCalderwood** [comment]
@rengelking `since` parameter is the tid rather than a timestamp.

**Comment by WillCalderwood** [comment]
@rengelking You're right. It still only filters for the last 60 transactions. I can't work out how to go beyond that.
